src/RegistrationController.js

```diff
@@ -198,6 +198,19 @@ export default BaseLoginController.extend({
         noCancelButton: true,
         title: loc('registration.form.title', 'login'),
         save: loc('registration.form.submit', 'login'),
+        parseErrorMessage: function (resp) {
+          const hasErrorCauses = resp.errorCauses && resp.errorCauses.length;
+
+          if (hasErrorCauses) {
+            const isDuplicateEmailCause = resp.errorCode === 'E0000001'
+              && resp.errorCauses[0].reason === 'UNIQUE_CONSTRAINT';
+
+            if (isDuplicateEmailCause) {
+              resp.errorCauses[0].errorSummary = loc('registration.error.userName.notUniqueWithinOrg', 'login');
```

Comment:
One use-case we need to cover is allowing developers to override this message. I think using the `login` bundle may prevent that, though. Is there a better way to allow this?

Reply:
I wanted to use the existing translations that exist on the widget but perhaps we need to move this outside.

Reply:
Why would you think using `login` bundle would prevent this. It's been used all over the place.

Reply:
I have the impression that the login bundle cannot be overridden through the user configuration, is that not the case?

Reply:
no. it meant to be override currently by the `i18n` config.